charts/k8ssandra/templates/grafana/configmap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.monitoring.grafana.provision_dashboards }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-k8ssandra-grafana-dashboards
+  labels:
+    grafana_dashboard: "true"
+{{ include "k8ssandra.labels" . | indent 4 }}
+data:
+  cassandra-condensed.json: |
+{{ .Files.Get "dashboards/cassandra-condensed.json" | indent 4 }}
+  cassandra-overview.json: |
```

Comment:
When I originally embedded these dashboards, I built a [script](https://github.com/k8ssandra/k8ssandra/blob/main/scripts/update-mcac-dashboards.sh) that automatically pulls updated versions from MCAC, invokes a [subscript](https://github.com/k8ssandra/k8ssandra/blob/main/scripts/templatize-dashboard.sh) for each that escapes them for Helm compatibility, adds a plugin declaration that's missing from MCAC's version of the Overview dashboard, and then drops the resulting files in place.

It seems like the escaping bit isn't needed anymore (🎉), and I see that you've included the plugin, but we should either update or delete the script.

Reply:
With this PR we don't need the generated k8s versions anymore. I'll update the script to pull the JSON files generated by jsonnet instead. Great catch!